tests/test_backends_qibotn.py

```diff
@@ -1,16 +1,18 @@
 import os
 
 import qibo
+from qibo.backends import _Global
 
 # Force quimb to use qibojit default number of threads.
+_Global.backend()
 os.environ["NUMBA_NUM_THREADS"] = f"{qibo.get_threads()}"
 from qibotn.backends.quimb import QuimbBackend
 
-from qibo.backends import GlobalBackend
+from qibo.backends import _Global
```

Comment:
The public API should be `get_backend()`, if you decided to prefix `Global` with a leading `_`.

So, the options are: either you drop the `_` or alternatively you use `get_backend()` (ok, in tests is acceptable the usage of private members, as written above - but this test in particular has no specific reason to behave differently than normal application code).

Reply:
Btw: why are you importing this for the second time in the same file?

Reply:
Updated it. Remove the usage of `_Global`.